### DISCUSSION TITLE
STT Request - Google Cloud

### DISCUSSION THREAD

**Discussion by JarbasAl** [OpenVoiceOS/ovos-buildroot#146] [Dev]
google cloud has been removed from ovos-core but available in mycroft-core, a plugin needs to be made and mapped to the old config string in the Factory class

related code:
- https://github.com/MycroftAI/mycroft-core/issues/3150
- https://github.com/OpenVoiceOS/ovos-plugin-manager/blob/dev/ovos_plugin_manager/stt.py#L86
- https://github.com/MycroftAI/mycroft-core/blob/dev/mycroft/stt/__init__.py#L117

**Comment by mikejgray** [comment]
Doesn't Neon have this implementation already?
